.goreleaser.yml

```diff
@@ -1,9 +1,65 @@
 builds:
-- 
+-
   goos:
   - windows
   - darwin
   - linux
   goarch:
     - amd64
-    - arm64
\ No newline at end of file
+    - arm64
+
+# .goreleaser.yaml
+release:
+  # Repo in which the release will be created.
+  # Default is extracted from the origin remote URL or empty if its private hosted.
+  github:
+    owner: great-fork
```

Comment:
This needs to be kvz, doesn't it?

Reply:
Yes, I forgot to change it while testing in my own repo.